pkg/pubsub/adapter/adapter.go

```diff
@@ -86,13 +85,6 @@ func (a *Adapter) Start(ctx context.Context) error {
 	return a.inbound.StartReceiver(ctx, a.receive)
 }
 
-func (a *Adapter) DeletePubSubSubscription(ctx context.Context) error {
```

Comment:
Did this move somewhere else, or was it dead code?

Reply:
this was removed from adapter, it was implemented and then replaced with PubSubBase, and this PR is an improvement to PubSubBase v1